expyfun/_input_controllers.py

```diff
@@ -440,3 +440,56 @@ def _point_in_tri(self, pos, tri):
             return True
         else:
             return False
+
+
+class CedrusBox(Keyboard):
+    """Class for Cedrus response boxes
+
+    Note that experiments with Cedrus boxes are limited to ~4 hours due
+    to the data type of their counter (milliseconds since start as integers).
+    """
+    def __init__(self, ec, force_quit_keys):
```

Comment:
Cool. We may want to add a warning to the log file or something so that this knowledge gets passed on, but looks good for now.